packages/ffe-form-react/src/Checkbox.js

```diff
@@ -30,7 +30,11 @@ const Checkbox = React.forwardRef((props, ref) => {
                 children(labelProps)
             ) : (
                 // eslint-disable-next-line jsx-a11y/label-has-for
-                <label {...labelProps}>{!hiddenLabel && children}</label>
+                <label {...labelProps}>
+                    <span className="ffe-checkbox__content">
```

Comment:
Trenger span her for att kunna bruka griden slik jag vill. Men kun inline elementer er ju lov i en label så dette burde ikke ha noen konsekvenser.